.circleci/config.yml

```diff
@@ -7,71 +7,18 @@ orbs:
 jobs:
   validate:
     docker:
-      - image: quay.io/giantswarm/opsctl
-        auth:
-          username: $DOCKER_USER
-          password: $DOCKER_PASS
+      - image: quay.io/giantswarm/architect
     steps:
       - checkout
       - run:
-          name: Validate KVM releases
-          command: opsctl validate releases -p kvm.yaml
-      - run:
-          name: Validate AWS releases
-          command: opsctl validate releases -p aws.yaml
-      - run:
-          name: Validate Azure releases
-          command: opsctl validate releases -p azure.yaml
-  prepare-helm-chart:
-    docker:
-      - image: bash:5.0.16
-    parameters:
-      app_name:
-        description: "Name of helm chart."
-        type: "string"
-      source:
-        description: "Platform releases YAML to deploy as helm chart."
-        type: "string"
-    steps:
-      - checkout
-      - run:
-          name: Create chart directory
-          command: mkdir -p helm/<< parameters.app_name >>/templates
-      - run:
-          name: Copy template release into chart templates
-          command: mv .circleci/release.yaml helm/<< parameters.app_name >>/templates
-      - run:
-          name: Copy releases to values.yaml
-          command: |
-            echo "versions:" > helm/<< parameters.app_name >>/values.yaml
-            sed 's/^/  /' << parameters.source >>.yaml >> helm/<< parameters.app_name >>/values.yaml
-      - run: 
-          name: Create Chart.yaml
-          command: |
-            cat \<<EOF > helm/<< parameters.app_name >>/Chart.yaml
-            apiVersion: v1
-            name: << parameters.app_name >>
-            version: [[ .Version ]]
-            appVersion: master
-            description: Giant Swarm releases for << parameters.source >>.
-            home: https://github.com/giantswarm/releases
-            EOF
-      - persist_to_workspace:
-          root: .
-          paths:
-            - ./helm
+          name: Validate releases
+          command: CGO_ENABLED=0 go test ./...
```

Comment:
Why not `architect/go-test`?

Reply:
I guess I didn't think about it since it's just the simple unit test. I guess it would be better though.